src/Codeception/Command/SelfUpdate.php

```diff
@@ -18,12 +23,17 @@
 class SelfUpdate extends Command
 {
     /**
-     * Class constants
+     * @var string
      */
     const NAME = 'Codeception';
+    /**
+     * @var string
+     */
     const GITHUB_REPO = 'Codeception/Codeception';
-    const PHAR_URL = 'http://codeception.com/';
-    const PHAR_URL_PHP56 = 'http://codeception.com/php56/';
+    /**
+     * @var string
+     */
+    const PHAR_URL = 'https://codeception.com/';
```

Comment:
Please set value to `https://codeception.com/php73/`  to distingush it from PHP 7.2 build of Codeception 4.